.github/workflows/build.yml

```diff
@@ -4,12 +4,17 @@ name: build
 on:
   push:
     branches:
-      - '**'
+      - "**"
     tags:
-      - 'v*.*.*'
+      - "v*.*.*"
   pull_request:
   schedule:
-    - cron: '0 10 * * *'  # everyday at 10am
+<<<<<<< HEAD
+    # everyday at 10am
+    - cron: "0 10 * * *"
+=======
+    - cron: "0 10 * * *"  # everyday at 10am
+>>>>>>> 9cea64da4a1379cdde3c509a09fc548f0da8b034
```

Comment:
You have committed a merge conflict.  Please resolve this.

Reply:
resolved with [d38e133](https://github.com/cisagov/domain-manager-ui/pull/247/commits/d38e1335f747335baa33f87d2745c9cf0cc84808)